### PR TITLE
Issue26

### PR DESCRIPTION
**Pull request by emicklei**

### CODE REVIEW

**Comment by peter-edge** [comment]
Can you please add a test that verifies the case that broke is fixed?

Here is the relevant break: https://github.com/emicklei/proto/issues/26#issuecomment-354897161

File: comment.go [hunk 1]
```diff
@@ -119,17 +126,24 @@ func takeLastComment(list []Visitee) (*Comment, []Visitee) {
...
+	esize := len(elements)
+	if esize == 0 {
+		return com
+	}
+	// last element must be a comment to merge
+	last, ok := elements[esize-1].(*Comment)
+	if !ok {
+		return com
+	}
+	// do not merge c-style comments
```
Comment (peter-edge) [inline comment.go]: Why? c-style comments can be valid.
  
```proto
int64 hello = 1; /* inline comment */
int64 foo = 2; /* inline
  multiline
  comment */
```

Comment (emicklei) [reply comment.go]: Would you want this 
```
/* 
one
two
*/
/*
three
four
*/
```
to be merged into 
/*
one
two
three
four
*/
```
? Clearly, c-style comments already allow multiline comments so I don't want to ignore the intention of the writer.
The scanner already reads all lines of the c-style comment.  I added an additional test to verify.

Comment (emicklei) [reply comment.go]: as c-style Comment values already have multiple lines, merging 2 Comment values is not what we want.

**Pull request merged**